utility/index.jsx

```diff
@@ -29,39 +33,86 @@ export function formatSeconds(input) {
   return time;
 }
 
-export const getPercentWin = (wins, games) => (games ? ((wins / games) * 100) : 0);
+export const getPercentWin = (wins, games) => (games ? (Math.round(((wins * 100) / games) * 100) / 100) : 0);
 
 export const camelToSnake = str =>
   str.replace(/\.?([A-Z]+)/g, (match, group) => `_${group.toLowerCase()}`).replace(/^_/, '');
 
 export const transformations = {
   hero_id: (row, col, field) => (
-    <div>
-      <img src={`${API_HOST}${constants.heroes[field] ? constants.heroes[field].img : ''}`} style={{ height: 24 }} role="presentation" />
-      <div className={styles.subText}>{constants.heroes[field] ? constants.heroes[field].localized_name : ''}</div>
+    <div style={{ position: 'relative' }}>
+      {row.version ?
+        <div style={{ position: 'absolute', left: -24, width: 2, height: '100%', backgroundColor: styles.linkColor }}>
+          <div data-tip data-for="parsed">
+            <ActionDoneAll color={styles.linkColor} style={{ position: 'relative', left: -10 }} />
+          </div>
+          <ReactTooltip id="parsed" place="top" type="light" effect="float">
+            Replay has been parsed for additional statistics
+          </ReactTooltip>
+        </div>
+      : null}
+      <img
+        src={`${API_HOST}${constants.heroes[field] ? constants.heroes[field].img : ''}`}
+        role="presentation"
+        style={{ height: 29, marginRight: 7 }}
+      />
+      <div style={{ display: 'inline-block', verticalAlign: `${row.player_slot ? 'top' : 'super'}` }}>
+        {constants.heroes[field] ? constants.heroes[field].localized_name : ''}
+        {row.match_id ?
+          <span className={styles.subText} style={{ display: 'block', marginTop: 1 }}>
+            {isRadiant(row.player_slot) ? 'Radiant' : 'Dire'}
+          </span> :
```

Comment:
should we color this Radiant/Dire text or is that too busy? I'm not sure it's even useful to distinguish on the player matches screen so it probably doesn't matter.